lib/sycamore/sycamore/transforms/detr_partitioner.py

```diff
@@ -542,16 +545,19 @@ def _run_text_extractor_document(
         use_cache: bool,
         use_ocr: bool,
         ocr_images: bool,
-        text_extractor_model: str,
+        text_extractor_model: Union[str, OcrModel],
```

Comment:
Can we make this an instance of `TextExtractor`? I could imagine wanting to configure PDFMiner similarly as well.

Reply:
Sorry for the quick merge. Let's update this as the need arises, it shouldn't cause any issues